js/piwik.js

```diff
@@ -4708,7 +4638,11 @@ if (typeof window.Piwik !== 'object') {
                 } else if (documentAlias.addEventListener) {
                     documentAlias.addEventListener('DOMContentLoaded', callback);
                 } else if (documentAlias.attachEvent) {
-                    documentAlias.attachEvent('onreadystatechange', callback);
+                    documentAlias.attachEvent('onreadystatechange', function () {
+                        if (documentAlias.readyState === 'complete') {
```

Comment:
`onreadystatechange` might be actually triggered twice. When it changes to interactive and when it changes to complete. We want to execute it only once.